dialog/file.go

```diff
@@ -745,6 +751,18 @@ func (f *FileDialog) Hide() {
 	}
 }
 
+// SetTitleText allows custom text to be set in the dialog title
+//
+// Since: 2.6
+func (f *FileDialog) SetTitleText(label string) {
+	f.titleText = label
+	if f.dialog == nil {
+		return
+	}
+	f.dialog.title.SetText(label)
+	f.dialog.win.Refresh()
```

Comment:
This line should not be needed - setting text will update visuals required.

Reply:
p.s. please try and keep methods in alphabetical order per-type (i.e. SetConfirmText would go before SetTitleText)

Reply:
I guess you're right consistent with what was there before works for this PR.